build.sbt

```diff
@@ -117,7 +124,8 @@ lazy val docs = project
   .settings(
     commonSettings,
     libraryDependencies ++= Seq(
-      "org.mockito" % "mockito-core" % MockitoVersion % Test
+      "org.mockito"       % "mockito-core" % MockitoVersion          % Test,
+      "org.scalatestplus" %% "mockito-3-2" % ScalatestMockitoVersion % Test,
```

Comment:
Here, too: we depend on 2.18.3, but I suspect this scalatestplus dependency pulls in 3.2 (though I haven't double-checked)

Reply:
Also doesn't appear to be a problem. Would suggest looking at mockito and selenium version alignment in a separate PR.

Reply:
that one seems safe: #253